api/grpcserver/v2alpha1/activation_test.go

```diff
@@ -28,9 +28,8 @@ func TestActivationService_List(t *testing.T) {
 	activations := make([]types.ActivationTx, 100)
 	for i := range activations {
 		atx := gen.Next()
-		vAtx := fixture.ToAtx(t, atx)
-		require.NoError(t, atxs.Add(db, vAtx, atx.Blob()))
-		activations[i] = *vAtx
+		require.NoError(t, atxs.Add(db, atx, types.AtxBlob{Blob: types.RandomBytes(100)}))
```

Comment:
NIT: I think the blob could be empty:
```suggestion
		require.NoError(t, atxs.Add(db, atx, types.AtxBlob{}))
```